src/Service.Host/client/src/components/RunMrp.js

```diff
@@ -22,7 +22,7 @@ function RunMrp() {
     const mrMasterLoading = useSelector(state =>
         itemSelectorHelpers.getItemLoading(state.mrMaster)
     );
-    const runMrpResult = useSelector(state => processSelectorHelpers.getData(state.runMrp));
+    //const runMrpResult = useSelector(state => processSelectorHelpers.getData(state.runMrp));
```

Comment:
this is changed on my branch. don't merge hell me

Reply:
I just commented it out to make the linter be quiet. I'll revert before I merge